src/lib/utils/scales-utils.js

```diff
@@ -51,6 +51,13 @@ const ORDINAL_SCALE_TYPE = 'ordinal';
 const CATEGORY_SCALE_TYPE = 'category';
 
 /**
+ * Literal scale.
```

Comment:
Might be worthwhile explaining how this differs from d3's identity scale

https://github.com/d3/d3-scale#scaleIdentity